packages/grafana-runtime/src/config.ts

```diff
@@ -238,16 +235,33 @@ function overrideFeatureTogglesFromLocalStorage(config: GrafanaBootConfig) {
   }
 }
 
-function overrideFeatureTogglesFromUrl(config: GrafanaBootConfig) {
+function overrideFeatureTogglesFromUrl(config: GrafanaBootConfig, allowWhiteListed = false) {
```

Comment:
Are we using the `allowWhiteListed` parameter at all?

```suggestion
function overrideFeatureTogglesFromUrl(config: GrafanaBootConfig) {
```

Reply:
oops, no. leftover